sources/packages/backend/libs/utilities/src/string-utils.ts

```diff
@@ -49,6 +49,9 @@ export function convertToASCII(rawContent?: string): Buffer | null {
   });
   const content = Buffer.from(rawContent, FILE_DEFAULT_ENCODING);
   for (const [index, char] of content.entries()) {
+    if (char < 32 && char !== 10 && char !== 13) {
+      content[index] = 63; // Replace with ? for control characters.
+    }
```

Comment:
Can we create the following constants to improve the code readability?
- NONPRINTABLE_CHARACTERS_LIMIT = 31
- CARRIAGE_RETURN = 13
- LINE_FEED = 10
- UNEXPECTED_CHAR = 63
_Note:_ please also user the `UNEXPECTED_CHAR ` at line 141.